addons/website_sale/models/website.py

```diff
@@ -410,7 +410,9 @@ class WebsiteSaleExtraField(models.Model):
     sequence = fields.Integer(default=10)
     field_id = fields.Many2one(
         'ir.model.fields',
-        domain=[('model_id.model', '=', 'product.template'), ('ttype', 'in', ['char', 'binary'])]
+        domain=[('model_id.model', '=', 'product.template'), ('ttype', 'in', ['char', 'binary'])],
+        required=True,
```

Comment:
It's probably better to set the required in the view on stable.
https://github.com/odoo/odoo/blob/dbc16551e716bb6660d6de0e14833cb96893acea/addons/website_sale/views/templates.xml#L1915

Reply:
sorry I'm not sure what you mean by set it on stable ?